lib/api/src/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -911,6 +910,9 @@ import qualified Cardano.Wallet.Address.Discovery.Sequential as Seq
 import qualified Cardano.Wallet.Address.Discovery.Shared as Shared
 import qualified Cardano.Wallet.Api.Types as Api
 import qualified Cardano.Wallet.Api.Types.Amount as ApiAmount
+import qualified Cardano.Wallet.Api.Types.Era as ApiEra
+    ( fromAnyCardanoEra
+    )
```

Comment:
Isn't this breaking our import style ?

Reply:
> Isn't this breaking our import style ?

We've been using this explicit import style in a number of places for a while now!

For example: https://github.com/cardano-foundation/cardano-wallet/blob/0f1bdf1f1702aa151c0950241fa82ba097d155ff/lib/wallet/src/Cardano/Wallet.hs#L866-L882

Reply:
frankly speaking I am not fond of this import style... I liked our previous style:
```
import Prelude

import SomeModule (f1, F(..))

import qualified SomeModule as SM
```
But I agree with Jonathan that there are a number of examples of
```
import qualified SomeModule as SM (f1, F(...))
```
already in our codebase

Reply:
> ```
> import Prelude
> 
> import SomeModule (f1, F(..))
> 
> import qualified SomeModule as SM
> ```

One problem is that we often import multiple modules into the same namespace. So we often end up with imports like:
```hs
import qualified SomeModule.Foo as SM
import qualified SomeModule.Bar as SM
```

Now when a reader sees some symbol `SM.wibble`, it's hard (at a glance) to see which module actually exports `wibble`.

We can make this clearer by writing:
```hs
import qualified SomeModule.Foo as SM
    ( wibble )
import qualified SomeModule.Bar as SM
    ( wobble )
```

IMO, a particularly egregious example of this is the practice of importing a ton of modules into the `W` namespace.